domain/domain.links.task.yml

```diff
@@ -1,5 +1,6 @@
-entity.domain.edit_form:
```

Comment:
It doesn't seem to affect anything. See the new PR in https://github.com/agentrickard/domain/pull/264